_posts/2022-08-03-WordPress-common-troubleshooting-scenarios.md

```diff
@@ -83,4 +83,4 @@ If your WordPress page is encountering application issues without the detailed e
  >![WordPress common troubleshooting scenarios](/media/2022/08/wp_home_site_url.png)
 2.	Disable all plugins by renaming the plugins folder to check if the redirection is being caused by any of the plugins. 
 3.	Please check if the WordPress admin URL https://<site home url>/wp-admin or any other inside URLs also facing too may redirection issue. 
```

Comment:
Please check if the WordPress admin URL https://<site home url>/wp-admin or any other URLs within the admin page are facing a redirection issue.